core/src/modules/configLoaders/team.ts

```diff
@@ -60,6 +62,23 @@ export async function loadTeam(
     { where: { ownerId: team.id } }
   );
 
+  // Set the cluster name from the team name and current environment if it
+  // hasn't already been changed.
+  const clusterNameSetting = await Setting.findOne({
+    where: { pluginName: "core", key: "cluster-name" },
+  });
+  if (
+    team.name &&
+    clusterNameSetting &&
+    clusterNameSetting.value === clusterNameSetting.defaultValue
+  ) {
+    const nodeEnv = env || "development";
+    const clusterName = `${team.name} - ${
+      nodeEnv.charAt(0).toUpperCase() + nodeEnv.slice(1)
+    }`; // the cluster name would be `Grouparoo - Development`
+    await clusterNameSetting.update({ value: clusterName });
+  }
+
```

Comment:
I don't feel too badly about the duplication, but I guess the /correct/ thing would be to DRY this and the version in `actions/team.ts` up.  I think the most relevant place to put this would be in a new `modules/ops/settings.ts` file - Ops (short for "operations") are where we've been putting business logic that's related to a model but transforms it and/or other models.  Then, you could test the new `SettingOps.nameClusterFromTeam()` method (or whatever you choose to call it) as well.